validator.js

```diff
@@ -85,7 +85,8 @@
       'el-GR': /^(\+30)?((2\d{9})|(69\d{8}))$/,
       'en-GB': /^(\+?44|0)7\d{9}$/,
       'en-US': /^(\+?1)?[2-9]\d{2}[2-9](?!11)\d{6}$/,
-      'en-ZM': /^(\+26)?09[567]\d{7}$/
+      'en-ZM': /^(\+26)?09[567]\d{7}$/,
+      'ru-RU': /^((8|\+7)(\s|-?))\(?\d{3}\)?(\s|-)?\d{1}(\s|-)?\d{1}(\s|-)?\d{1}(\s|-)?\d{1}(\s|-)?\d{1}(\s|-)?\d{1}(\s|-)?\d{1}$/
```

Comment:
This could be simplified:
- `\d{1}` is the same as `\d`
- you could replace `(\s|-)` with `[\s-]`
- you could replace the seven repeated `(\s|-)?\d{1}` patterns with a single `([\s-]?\d){7}`